src/styles/stars.css

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url('../images/star-active.svg') no-repeat center center/contain;
```

Comment:
Consider using SCSS variables for the URL paths to the images to ensure consistency and ease of updates.